src/library/Uno.Material/Extensions/ControlExtensions.cs

```diff
@@ -73,6 +77,63 @@ public static class ControlExtensions
 		[DynamicDependency(nameof(GetAlternateContent))]
 		public static void SetAlternateContent(Control obj, object value) => obj.SetValue(AlternateContentProperty, value);
 
-#endregion
+		#endregion
+
+		#region DependencyProperty: Elevation
+
+		/// <summary>
+		/// Gets or sets the level of elevation to depict for the attached view. In Material, elevation can be used to drive both the shadow and the surface tint color (https://m3.material.io/styles/elevation/overview)
+		/// </summary>
+		public static DependencyProperty ElevationProperty { [DynamicDependency(nameof(ElevationProperty))] get; } = DependencyProperty.RegisterAttached(
+			"Elevation",
+			typeof(int),
+			typeof(ControlExtensions),
+			new PropertyMetadata(default, OnElevationChanged));
```

Comment:
Do not use `default` for default property value unless it is a reference type. The ctor here takes in an `object` and you will be passing null, causing `GetElevation` to throw, if not previously set.
```suggestion
			new PropertyMetadata(default(int), OnElevationChanged));
```

Reply:
but, however, be aware of unoplatform/uno#11448